### PR TITLE
edit mongodb-driver getCollection method

### PR DESCRIPTION
**Pull request by alexZogich**
edit mongodb-driver getCollection  method to return collection instead of single value

### CODE REVIEW

**Comment by alexZogich** [comment]
@kucherenko @kolesnik 

I suggest getCollection method to be updated:

```javascript

    getCollection(type, date, period) {

        return this.db.collection(this.collection).find({
            period: childPeriod(period),
            startDate: {
                $gte: date
            },
            dateKey: new RegExp(type + '\\b')
        }).sort({ startDate: 1 });
    }
```

Maybe its better to name this method another way, and even to make it more clear: 

```javascript
...
        return this.db.collection(this.collection).find({
            period: period,
            startDate: {
...
```

File: src/mongodb-driver.js [hunk 1]
```diff
@@ -48,8 +49,17 @@ export class MongoDbDriver {
...
 
     getCollection(type, date, period) {
-        let dateKey = this._getDateKey(type, date, period);
-        return this.db.collection(this.collection).find({period: period, dateKey: dateKey});
+        const periodEnd = moment(date);
+        const periodStart = moment(date).subtract(1, period);
+
+        return this.db.collection(this.collection).find({
+            period: childPeriod(period),
+            startDate: {
```
Comment (kolesnik) [inline src/mongodb-driver.js:53]: please add index for "startDate" in _ensureIndex()

Comment (alexZogich) [reply src/mongodb-driver.js:53]: Done

**Comment by kolesnik** [comment]
+1, I am fine with current name it might be "getList"

**Pull request merged**